### PR TITLE
copy !q to !help

### PR DESCRIPTION
**Pull request by Mikaela**
People trying to get help often use !help and then get confused by the
"Need any help" reply. I think better is to use the !q reply so the
people know they could improve their question.

I don't know regexp enough well to use the same code for !q if that is
even possible.

### CODE REVIEW

**Comment by Mikaela** [comment]
This happened today with martinus__.

File: znclinker.pm [hunk 1]
```diff
@@ -94,8 +94,9 @@ sub OnChanMsg {
 	if ($what=~/^!win/) {
 		$self->put_chan($chan=>'Installing ZNC on Windows: http://wiki.znc.in/Installation#Cygwin');
 	}
-	if ($what eq '!help') {
-		$self->put_chan($chan=>'Need any help?');
+	if (my ($to) = $what=~/^!help(?:\s+(\S+))?/i) {
+		$to = $to // $nick;
+		$self->put_chan($chan=>"$to, we are not telepaths, please ask a concrete question and wait for an answer. Be sure that you have checked http://wiki.znc.in/FAQ first. You may want to read http://catb.org/~esr/faqs/smart-questions.html");
```
Comment (DarthGandalf) [inline znclinker.pm]: Don't copy paste the message. If later someone changes it in one place, certainly the other one will be forgotten.

Comment (Mikaela) [inline znclinker.pm]: Would you have something against performing tabs to four spaces conversion? My Vim isn't fan of your way...

Or do you have something I could add as modeline?

Where exactly should I add these variables? http://www.cgi101.com/class/ch2/text.html

Comment (DarthGandalf) [inline znclinker.pm]: Such conversion should be a separate commit, without any other change, except addition of modeline.
My vim settings are
```vim
set ts=4
set sw=4
set softtabstop=4
```
I think it's translateable to modeline somehow.
